src/m4b_merge/m4b_helper.py

```
@@ -462,34 +646,53 @@ def fix_chapters(self):
+        else:
+            logging.info("Using mutagen for chapter operations (mp4chaps not available)")
+            # Parse chapter file
+            chapter_markers = self._parse_chapter_file(chapter_file)
+            if not chapter_markers:
+                logging.error("Chapter file is empty, attempting to correct")
+                # Try to repair by using existing chapters from the file
+                try:
+                    mp4 = MP4(m4b_to_modify)
+                    if mp4.chapters:
+                        chapter_markers = [(str(chapter.start), chapter.title) for chapter in mp4.chapters]
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)